packages/@dcl/inspector/src/lib/babylon/decentraland/sdkComponents/gltf-container.ts

```diff
@@ -125,6 +124,13 @@ async function tryLoadGltfAsync(sceneId: string, entity: EcsEntity, filePath: st
     (assetContainer) => {
       processGLTFAssetContainer(assetContainer, entity)
 
+      // remove old entities
+      const prevChildren = entity.getChildren()
+      for (const child of prevChildren) {
+        child.setEnabled(false)
+        child.dispose(false, true)
+      }
```

Comment:
The issue was that when the scene is reloaded, the gltfs are loaded from two places, a side effect in the Renderer.tsx component, and the CRDT messages coming from the engine. I dunno what is the reason for the side effect in the Renderer component but by looking at the blame it came to fix an issue with the undo/redo, so I didn't remove it, but this caused GLTFs to be loaded twice when the scene is reloaded for every GTLFContainer. What I did was adding the piece above that empties an entity when loading a GLTF into it, so if it's loaded twice, the second time will clear the previous one. This previous load was the one that remained after deleting the entity causing the problem described in the issue.